docs/sql-guide/functions/function-euclidean-distance.md

```diff
@@ -0,0 +1,32 @@
+---
+title: EUCLIDEAN_DISTANCE()
+layout: default
+parent: Functions
+grand_parent: SQL guide
+---
+# EUCLIDEAN_DISTANCE() function
+
+`EUCLIDEAN_DISTANCE()` is a mathematical function that can be performed on `VECTOR` data type columns.
+The Euclidean (or Euclidian) distance between two vectors is the square root of the sum of the squares
+the pairwise differences between the vectors. This is the distance metric most commonly used when talking
+about distances between locations on a plane.
+
+## Syntax
+
+```sql
+EUCLIDEAN_DISTANCE (<vector1>, <vector2>)
```

Comment:
Same as above, could this be `EUCLIDEAN_DISTANCE ([<vector-literal>|<vector-col>],...)` :D